docs/technical/conflicts.md

```diff
@@ -10,32 +10,40 @@ For example, if you merge two branches in a repo, there may be conflicting
 changes between the two branches. Most DVCSs require you to resolve those
 conflicts before you can finish the merge operation. Jujutsu instead records
 the conflicts in the commit and lets you resolve the conflict when you feel like
-it. 
-
+it.
 
 ## Data model
 
-When a merge conflict happens, it is recorded within the tree object as a
-special conflict object (not a file object with conflict markers). Conflicts
-are stored as a lists of states to add and another list of states to remove. A
-"state" here can be a normal file, a symlink, or a tree. These two lists
-together can be a viewed as a simple algebraic expression of positive and
-negative terms. The order of terms is undefined.
-
-For example, a regular 3-way merge between B and C, with A as base, is `B+C-A`
-(`{ removes=[A], adds=[B,C] }`). A modify/remove conflict is `B-A`. An add/add
-conflict is `B+C`. An octopus merge of N commits usually has N positive terms
-and N-1 negative terms. A non-conflict state A is equivalent to a conflict state
-containing just the term `A`. An empty expression indicates absence of any
-content at that path. A conflict can thus encode a superset of what can be
-encoded in a regular path state.
-
+When a merge conflict happens, it is recorded as an ordered list of tree objects
+linked from the commit (instead of the usual single tree per commit). There will
+always be an odd number of trees linked from the commit. You can think of the
+first tree as a start tree, and the subsequent pairs of trees to apply the diff
+between onto the start. Examples:
+
+* If the commit has trees A, B, C, D, and E it means that the contents should be
```

Comment:
Optionally, you could add:

> On disk, the [storage format](git.compatibility.md#format-mapping-details) is currently slightly different. A tree for a positive term is called a "side" and a tree for a negative term is called a "base".

This could be inside the list, or as a separate paragraph after the list.

Reply:
I added a link at the end of this section, and I also updated `git-compatibility.md` to say that the `.jjconflict-` directories are not authoritative.